src/components/NotesList.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from "react";
+import List from "@mui/material/List";
+import PropTypes from "prop-types";
+import Alert from "@mui/material/Alert";
+import { CircularProgress } from "@mui/material";
+import { useQuery } from "react-query";
+import { DragDropContext, Droppable, Draggable } from "react-beautiful-dnd";
+import InfiniteScroll from "react-infinite-scroll-component";
+
+import MESSAGES from "../../src/config/constants/messages";
+
+import Note from "./Note";
+import styles from "../pages/styled";
+
+const Notes = ({ userId, noteChosen }) => {
+  const { data, isSuccess, isLoading } = useQuery("shared", () =>
+    fetch("https://mocki.io/v1/6e70ca5e-cb79-4b2f-8c99-8b99b08eb542").then(
+      (res) => res.json()
+    )
+  );

```

Comment:
Move this request to the api/notes (also please update promise with async/await!)
It will be:
```
export const useGetSharedNotes = () => useQuery("shared", () =>
    fetch("https://mocki.io/v1/6e70ca5e-cb79-4b2f-8c99-8b99b08eb542").then(
      (res) => res.json()
    )
  );
```

And in container:
`const [ data, isSuccess, isLoading ] = useGetSharedNotes();`